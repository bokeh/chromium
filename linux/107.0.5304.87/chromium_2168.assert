type: account-key
authority-id: canonical
revision: 2
public-key-sha3-384: BWDEoaqyr25nF5SNCvEv2v7QnM9QsfCc0PBMYD_i2NGSQ32EF2d4D0hqUel3m8ul
account-id: canonical
name: store
since: 2016-04-01T00:00:00.0Z
body-length: 717
sign-key-sha3-384: -CvQKAwRQ5h3Ffn10FILJoEZUXOv6km9FwA80-Rcj-f-6jadQ89VRswHNiEB9Lxk

AcbBTQRWhcGAARAA0KKYYQWuHOrsFVi4p4l7ZzSvX7kLgJFFeFgOkzdWKBTHEnsMKjl5mefFe9ji
qe8NlmJdfY7BenP7XeBtwKp700H/t9lLrZbpTNAPHXYxEWFJp5bPqIcJYBZ+29oLVLN1Tc5X482R
vCiDqL8+pPYqBrK2fNlyPlNNSum9wI70rDDL4r6FVvr+osTnGejibdV8JphWX+lrSQDnRSdM8KJi
UM43vTgLGTi9W54oRhsA2OFexRfRksTrnqGoonCjqX5wO3OFSaMDzMsO2MJ/hPfLgDqw53qjzuKL
Iec9OL3k5basvu2cj5u9tKwVFDsCKK2GbKUsWWpx2KTpOifmhmiAbzkTHbH9KaoMS7p0kJwhTQGA
o9aJ9VMTWHJc/NCBx7eu451u6d46sBPCXS/OMUh2766fQmoRtO1OwCTxsRKG2kkjbMn54UdFULl9
VfzvyghMNRKIezsEkmM8wueTqGUGZWa6CEZqZKwhe/PROxOPYzqtDH18XZknbU1n5lNb7vNfem9F
2ai+3+JyFnW9UhfvpVF7gzAgdyCqNli4C6BIN43uwoS8HkykocZS/+Gv52aUQ/NZ8BKOHLw+7ant
Q0o8W9ltSLZbEMxFIPSN0stiZlkXAp6DLyvh1Y4wXSynDjUondTpej2fSvSlCz/W5v5V7qA4nIcG
vUvV7RjVzv17ut0AEQEAAQ==

AcLDXAQAAQoABgUCV83k9QAKCRDUpVvql9g3IBT8IACKZ7XpiBZ3W4lqbPssY6On81WmxQLtvsMV
WTp6zZpl/wWOSt2vMNUk9pvcmrNq1jG9CuhDfWFLGXEjcrrmVkN3YuCOajMSPFCGrxsIBLSRt/bP
nrKykdLAAzMfG8rP1d82bjFFiIieE+urQ0Kcv09Jtdvavq3JT1Tek5mFyyfhHNlQEKOzWqmRWiLg
3c3VOZUs1ZD8TSlnuq/x+5T0X0YtOyGjSlVxk7UybbyMNd6MZfNaMpIG4x+mxD3KHFtBAC7O6kLe
eX3i6j5nCY5UABfA3DZEAkWP4zlmdBEOvZ9t293NaDdOpzsUHRkoi0Zez/9BHQ/kwx/uNc2WqrYm
inCmu16JGNeXqsyinnLl7Ghn2RwhvDMlLxF6RTx8xdx1yk6p3PBTwhZMUvuZGjUtN/AG8BmVJQ19
rsGSRkkSywvnhVJRB2sudnrMBmNS2goJbzSbmJnOlBrd2WsV0T9SgNMWZBiov3LvU4o2SmAb6b+k
rYwh8H5QHcuuYJuxDjFhPswIp6Wes5T6hUicf3SWtObcDS4HSkVS4ImBjjX9YgCuFy7QdnooOWEY
aPvkRw3XCVeYq0K6w9GRsk1YFErD4XmXXZjDYY650MX9v42Sz5MmphHV8jdIY5ssbadwFSe2rCQI
6UX08zy7RsIb19hTndE6ncvSNDChUR9eEnCm73eYaWTWTnq1cxdVP/s52r8uss++OYOkPWqh5nOu
haRn7INjH/yZX4qXjNXlTjo0PnHH0q08vNKDwLhxS+D9du+70FeacXFyLIbcWllSbJ7DmbumGpFo
yYbtj3FDDPzachFQdIG3lSt+cSUGeyfSs6wVtc3cIPka/2Urx7RprfmoWSI6+a5NcLdj0u2z8O96
HxeIgxDpg/3gT8ZIuFKePMcLDM19Fh/p0ysCsX+84B9chNWtsMSmIaE57V+959MVtsLu7SLb9gi7
skrju0pQCwsu2wHMLTNd1f3PTHmrr49hxetTus07HSQUApMtAGKzQilF5zqFjbyaTd4xgQbd+PKW
CjFyzQTDOcUhXpuUGt/IzlqiFfsCsmbj2K4KdSNYMlqIgZ3Azu8KvZLIhsyN7v5vNIZSPfEbjdeu
ClU9r0VRiJmtYBUjcSghD9LWn+yRLwOxhfQVjm0cBwIt5R/yPF/qC76yIVuWUtM5Y2/zJR1J8OFq
qWchvlImHtvDzS9FQeLyzJAOjvZ2CnWp2gILgUz0WQdOk1Dq8ax7KS9BQ42zxw9EZAEPw3PEFqRy
IQsRTONp+iVS8YxSmoYZjDlCgRMWUmawez/Fv5b9Fb/XkO5Eq4e+KfrpUujXItaipb+tV8h5v3tr
oG3Ie3WOHrVjCLXIdYslpL1O4nadqR6Xv58pHj6k

type: snap-declaration
format: 4
authority-id: canonical
revision: 17
series: 16
snap-id: XKEcBqPM06H1Z7zGOdG5fbICuf8NWK5R
plugs:
  audio-record:
    allow-auto-connection: true
  bluez:
    allow-auto-connection: true
  browser-support:
    allow-auto-connection: true
  camera:
    allow-auto-connection: true
  cups-control:
    allow-auto-connection: true
  joystick:
    allow-auto-connection: true
  personal-files:
    allow-auto-connection: true
    allow-installation:
      -
        plug-attributes:
          read: \$HOME/\.config/chromium
      -
        plug-attributes:
          read: \$HOME/\.(config/chromium|chromium-browser\.init)
  removable-media:
    allow-auto-connection: true
  system-files:
    allow-auto-connection: true
    allow-installation:
      plug-attributes:
        read: /etc/chromium-browser/policies
      plug-names:
        - etc-chromium-browser-policies
  system-packages-doc:
    allow-auto-connection: true
  u2f-devices:
    allow-auto-connection: true
publisher-id: canonical
snap-name: chromium
timestamp: 2020-10-15T12:24:48.459507Z
sign-key-sha3-384: BWDEoaqyr25nF5SNCvEv2v7QnM9QsfCc0PBMYD_i2NGSQ32EF2d4D0hqUel3m8ul

AcLBUgQAAQoABgUCX4g/kAAAJf8QAMj4bVY/ZgTdtB01Ew4kgKxQE/6BstI7lQq1n32J6RlcB+7/
0+Q3mqiZH6fOozxbWYpXL4UtzHl7alh2mu7N35mqP98ul8PEfgol8YP2HKJhO7HGmnnCxXawtgc9
LhGSqQu5Smwf4+3fI2BIWatqv/Oyrs2xfdh1Z8fxms0oQkpspEcOU/hGtMNf8YMJYnnoLAfWuY7Z
kXDmtdPrH/UCNS8I3XEtDQOzkMWDmS3sCnJZgifsmmTioHuMECzbQENt+ho2jc1P+09h/KglDGOI
AzRp91bvWrlxYje+l1yRZk/FVAOaUov2SI3n5ZglYkdH4Ecrd8MBqYg6q6Lh2tCxt43DGZDkQY+D
CE7Iv/DHXGQUpncVtv3A8fIgEp4nmpGnN97HZ/ZOROi26tq5ol9zog5VUH2ZRi22yWJlKyrysoEH
oHMyArrCti56c7Xeh2FzzBgAsDyOGcznJlYXE/gfuleSNReoUURYDfZUDKf5fQnDMhYXbFOzEik0
6hpAORz23DN0O1R65Tl7ICNfw0TM00no0TVI58Zi7MCxnwUustdj/jrsKBrL8163MD/ll3XOTg/y
lQEU4hhrZkYfYoDViVkwggi9iott6E23z/3m7g2tY/oBmjNSsVWqH4t6ZHCga33FTkff86DoOG+3
Ha8E/+PaECCWrgBlwBekfrK7RQWh

type: snap-revision
authority-id: canonical
snap-sha3-384: Ve9AK2PMVvCxzqUeJAiP3lmcXG0ifo5F5_fC2AoaeXn9UbThaPM08nE3ZqJSi0Bd
developer-id: canonical
provenance: global-upload
snap-id: XKEcBqPM06H1Z7zGOdG5fbICuf8NWK5R
snap-revision: 2168
snap-size: 152592384
timestamp: 2022-10-28T12:37:01.018502Z
sign-key-sha3-384: BWDEoaqyr25nF5SNCvEv2v7QnM9QsfCc0PBMYD_i2NGSQ32EF2d4D0hqUel3m8ul

AcLBUgQAAQoABgUCY1vM7QAAhtgQAE0VenUvjUqAHVEXwb6yrqYaAjlkReJ//M1Xm6C3xKGGPwO2
eLZbyCSaeBGbGzl+Pv9qPGWpfFvHkiKH2ITyY/Pb88DRPuql5HKQ9B1fC6dP+9q2O7BnwsT6H+iI
fqaE25XQJK6AjMgrWTDAX+QZajECxzFL3IbVRNVljEJZpulbfgBsg0Jc9LICKFTygnd+cqcNWANS
Q0VVccwAWze5GwfCba+Kw/1MCU3OOO4MjvrlJJ78mUGEWVpDRy7uRhWA5+JjupmDd0uA14teYGQ/
/Giuje+Hd0EhprfZYUWCDwmg+A+D0gijL1fmKVzrKKzH/IsLTC7TOWZgBMizeIyp4WdW+tlioeNw
zag6OMGa0Hk33/+6ghuT8OOaQHfKloUL1PWUGZP8kvbGujVXL+bRU5zOpkzCI2AGNjsKuwDHUYcy
cIHHVRtES4g9IP0MZ/DpbP9gWXQUVXTfiGwfsSR1soHdeF/s1eVIxKEISDy9KI6y2pRLtoNhULXP
M1wH0T3wOHwh/Wf9MtCvNgC83KRwtKfd+azjXurXQ37gLmCBtAWIav4I/b20h+VgDJogawQfWL5l
h42FlBSn+VMD+7sSWOuhYVKo04j4kAxhLyTsgnlebcTlRMvfdOE4tFX7ALQpo7baurYQ0ENb8Qo1
i0OZRM0M3pQdd5XZkEKpB4pThzWo
