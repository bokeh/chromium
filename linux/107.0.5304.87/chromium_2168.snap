version https://git-lfs.github.com/spec/v1
oid sha256:07268c98f6df84eea13023f3ba2c719bff3e96a03acc113367cb897b0f4c25cc
size 152592384
